### DISCUSSION TITLE
The plugin disappeared when translating on the right click of a web page. 网页右键时翻译时发现该插件消失了

### DISCUSSION THREAD

**Discussion by stuffren** [immersive-translate/immersive-translate#1457] [Help]
### 概述 | Summary

开发者，你好！我用这个插件有段时间了，很不错！！但是今天看英文网页右键时发现该插件消失了，就进入到谷歌扩展管理，它提示该插件已损坏，然后重装了之后，发现无法仍然翻译。
Hello developer! I've been using this plugin for a while and it's great! But today when I look at the English web page right click and found that the plugin disappeared, so I went to Google Extension Manager, it suggests that the plugin is damaged, then after reinstalling it, I found that I can't still translate.

### 补充信息 | Additional information

重装之后仍然无法翻译
After reinstalling it, it still doesn't translate

**Comment by keyboard3** [comment]
https://dash.immersivetranslate.com/  打开它重置一下设置看

**Reply by stuffren** [reply]
Thanks for the reply. After I gave you feedback on the earlier one, after a while I didn't do anything and it translated properly again!
感谢你的回复。刚才的我把情况反馈给你之后，过了一会儿，我什么也没有操作，它又可以正常翻译了！
One question I wanted to check with you, after reinstalling the version, it has a gray icon on the right side of the page. Is this a feature of the new version?
It's a nice addition, it's much more convenient than going to click on it in the right click!
有个问题想和你确认一下，重装之后的版本，它有一个灰色的图标在网页的右侧。这是新版本的特性吗？
这个功能加得好，比在右键里面去点击它会更加便捷了！

**Reply by keyboard3** [reply]
是的

**Reply by stuffren** [reply]
OKK，3Q!!
